mappings/com/mojang/blaze3d/systems/RenderCallStorage.mapping

```diff
@@ -1,2 +1,9 @@
 CLASS net/minecraft/class_4491 com/mojang/blaze3d/systems/RenderCallStorage
+	COMMENT A storage of render calls with recording and processing states. It exposes
+	COMMENT three queues: a recording queue, a processing queue, and a last processed
+	COMMENT queue. The recording queue is equal to the processing queue during processing
+	COMMENT and different during recording.
```

Comment:
these are the info post-revelation, and back then we only had recording queue ever appearing